src/clean_air/util/file_converter.py

```diff
@@ -38,11 +41,73 @@ def generate_dataframe(filepath):
         # that into a pandas dataframe.
         temp_dataset = xr.open_dataset(filepath)
         temp_dataframe = temp_dataset.to_dataframe()
+    elif filepath.endswith(".csv"):
+        # Not only do we need to read this format into a dataframe, but we
+        # also need to reformat it into acceptable standards before passing
+        # it on to saving functions.
+        temp_dataframe = csv_reformatter(filepath)
     else:
         raise ValueError("No reader configured yet for this input format.")
     return pd.DataFrame(temp_dataframe)
 
 
+def csv_reformatter(filepath):
+    """This function uses a template to reformat known-source CSV files into a
```

Comment:
Given how closely this is tied to the AURN CSV data, I feel like that should be made clearer. I'm not sure this would work for other CSV data, for example?

Reply:
Yes, you are quite right.  I will make this more specific.